routing/routes.go

```diff
@@ -48,6 +48,18 @@ func NewRoutes() (r *Routes) {
 			"/api/namespaces/{namespace}/services/{service}/metrics",
 			handlers.ServiceMetrics,
 		},
+		{
+			"GraphNamespace",
+			"GET",
+			"/api/namespaces/{namespace}/graphs/{vendor}",
+			handlers.GraphNamespace,
+		},
+		{
+			"GraphService",
+			"GET",
+			"/api/namespaces/{namespace}/services/{service}/graphs/{vendor}",
```

Comment:
I think the vendor should be supplied as a query param and if not provided should default to 'cytoscape'.